public/js/data.js

```diff
@@ -97,6 +97,13 @@ function updatingNews() {
             `
             newsArticles.append(html);
         }
+    });
+}
 
+function updatingLatestPrice() {
+    let latestPrice = $("#latestPriceDisplay");
+    latestPrice.empty
+    $.get("/api/latestPrice/" + ticker, response => {
+        latestPrice.append(response);
```

Comment:
since you have a priceChart, do you mind adding this function to priceChart? Because data.js is getting crowded
Thanks!